public/pl-static/css/syntax.css

```diff
@@ -0,0 +1,65 @@
+.highlight .hll { background-color: #ffc; }
```

Comment:
NOTE: vendor-ish default styling for Pygments (http://jekyllrb.com/docs/templates/#code-snippet-highlighting) for use with our code examples.